components/ModDashboardLeftNavbar.vue

```diff
@@ -1,64 +1,56 @@
-                <div class="flex items-center justify-center w-1/5">
-                    <component :is="submissionListEntry.svgIcon" class="h-5" />
-                </div>
-                {{ $t(submissionListEntry.displayText) }} ({{ submissionListEntry.count }})
-            </button>
+            <div class="flex flex-row w-4/5 justify-start items-center border-b-2 border-slate-200">
+                <SVGNoteStackAddSvg class="h-4" />
+                <button data-testid="mod-dashboard-leftnav-for-review"
+                    @click="updateSubmissionListViewState('forReview')"
+                    class="flex flex-row items-center text-start p-1">
+                    {{ $t("modDashboardLeftNav.forReview") }} ({{ 0 }})
+                </button>
+            </div>
+            <div class="flex flex-row w-4/5 justify-start items-center border-b-2 border-slate-200">
+                <SVGCheckBoxSvg class="h-5" />
+                <button data-testid="mod-dashboard-leftnav-approved" @click=" updateSubmissionListViewState('approved')"
+                    class="flex flex-row items-center text-start p-1 ">
+                    {{ $t("modDashboardLeftNav.approved") }} ({{ 0 }})
+                </button>
+            </div>
+            <div class="flex flex-row w-4/5 justify-start items-center ">
+                <SVGDisabledByDefault class="h-4" />
+                <button data-testid="mod-dashboard-leftnav-rejected" @click="updateSubmissionListViewState('rejected')"
+                    class="flex flex-row items-center text-start p-1">
+                    {{ $t("modDashboardLeftNav.rejected") }} ({{ 0 }})
+                </button>
+            </div>
         </div>
     </div>
 </template>
 
 <script setup lang="ts">
 import { ref, Ref } from 'vue'
-import NoteStackAddSvg from "../assets/icons/note-stack-add.svg"
-import CheckBoxSvg from "../assets/icons/check-box.svg"
-import DisabledByDefault from "../assets/icons/disabled-by-default.svg"
-
-const modDashboardViewOptions = [{
-    displayText: "Facilities",
-    value: "facilities"
-},
-{
-    displayText: "Healthcare Professionals",
-    value: "healthcareProfessionals"
-}]
+import SVGNoteStackAddSvg from "../assets/icons/note-stack-add.svg"
+import SVGCheckBoxSvg from "../assets/icons/check-box.svg"
+import SVGDisabledByDefault from "../assets/icons/disabled-by-default.svg"
 
-const submissionListViewOptions = [{
-    displayText: "For Review",
-    value: "forReview",
-    svgIcon: NoteStackAddSvg,
-    count: 0,
-},
-{
-    displayText: "Approved",
-    value: "approved",
-    svgIcon: CheckBoxSvg,
-    count: 0,
-},
-{
-    displayText: "Rejected",
-    value: "rejected",
-    svgIcon: DisabledByDefault,
-    count: 0,
-}]
 
-const selectedSubmissionListView: Ref<String> = ref(submissionListViewOptions[0].value)
-const selectedDashboardView: Ref<String> = ref(modDashboardViewOptions[0].value)
+const selectedSubmissionListView: Ref<string> = ref("forReview")
```

Comment:
future improvement: can  make these strings an enum for visibility and preventing typos